wsagent/che-core-git-impl-jgit/src/test/java/org/eclipse/che/git/impl/jgit/JGitConnectionTest.java

```diff
@@ -145,4 +148,20 @@ public void shouldDoNothingWhenTransportHttpReceived() throws Exception{
         //then
         verifyZeroInteractions(transportHttp);
     }
+
+    /**
+     * Check branch using current repository reference is returned
+     * @throws Exception if it fails
+     */
+    @Test
+    public void checkBranchName() throws Exception {
```

Comment:
Hum method name is subjective. It actually checks something so I can use check in it

Also it's divided in given/then I don't see any improvement on adding //given on mock() call